Eatery/Controllers/Eateries/CampusEateriesViewController.swift

```diff
@@ -51,6 +51,8 @@ class CampusEateriesViewController: EateriesViewController {
             self.updateState(.presenting, animated: true)
 
             self.pushPreselectedEateryIfPossible()
+
+            print("Successfully loaded \(eateries.count) campus eateries")
```

Comment:
Do we still want to keep this print statement?

Reply:
I at least added it with the intention of keeping it.